docs/docs/reference/production-deployment.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Production Deployment
+description: >-
+  This article covers production deployment requirements and concerns
+---
+
+# Production Deployment
+
+This page covers the topic of running Pomerium in a production configuration.  See the [quick start section](../quick-start/) for canned example configurations.  
+
+Please also see [architecture](../#architecture) for information on component interactions.
+
+## Service Mode
+
+### All in One
+
+In smaller deployments or while testing, it may be desirable to run in "all-in-one" mode.  This reduces resource footprint and simplifies DNS configuration.  All URLs point at the same pomerium service instance.
+
+### Dedicated Services
+
+In larger footprints, it is recommended that you run Pomerium as a collection of independent services.  This limits blast radius in the event of vulnerabilities and allows for independing [scaling](#scaling) and monitoring of services.
+
+In more complicated deployments, you may also choose to use a single Authorize and/or Authenticate cluster to service multiple proxy clusters.
+
+## Scaling
+
+In dedicated service mode, you have the opportunity to scale the components of Pomerium independently.
```

Comment:
> In **dedicated** service mode

hmmm dedicated makes me think a server that does one thing, vs separated services.... Though I'm sure of a better word or way to describe it.... what about "If running each component of Pomerium separately as a distinct service,"? What do you think?

Reply:
Fair point.  What about `discrete`?  Or we could adopt monolith-microservice convention.

Reply:
`Discrete` sounds great.